.github/workflows/release.yml

```diff
@@ -56,6 +56,14 @@ jobs:
       - build
     runs-on: ubuntu-latest
     timeout-minutes: 10
+    steps:
+      - name: Generate token
+        id: generate_token
+        uses: tibdex/github-app-token@v1
+        with:
+          app_id: ${{ secrets.OS_GITHUB_APP_ID }}
+          private_key: ${{ secrets.OS_GITHUB_APP_PRIVATE_KEY }}
```

Comment:
`${{ secrets.GITHUB_TOKEN }}` below should be `${{ steps.generate_token.outputs.token }}` so it will use the generated token